odxtools/multiplexer.py

```diff
@@ -236,3 +236,33 @@ def decode_from_pdu(self, decode_state: DecodeState) -> ParameterValue:
         decode_state.origin_byte_position = orig_origin
 
         return result
+
+    @override
+    def get_static_bit_length(self) -> int | None:
+        """
+        Returns the static bit length of the multiplexer structure, if determinable.
+
+        If all cases (including the default, if present) have the same static bit length,
+        the codec length is considered static and is returned.
+        Otherwise, returns None to indicate that the size is dynamic.
+        """
+        reference_case = self.default_case if self.default_case else self.cases[0]
+        if reference_case.structure is None:
+            return None
+        case_bit_length = reference_case.structure.get_static_bit_length()
```

Comment:
shouldn't the length of cases that don't define a structure be assumed to be zero, i.e.,
```suggestion
       if reference_case.structure is None:
            case_bit_length = 0
        else:
            case_bit_length = reference_case.structure.get_static_bit_length()
```

(analogous change required in the loop. this is quite an edge case because it will only return 0 if *all* cases are empty thus making the multiplexer pointless, but this would not be the worst thing I have seen in this industry...)

Reply:
done